README.md

```diff
@@ -29,11 +29,25 @@ There is also an experimental binary which uses Tessera's POSIX storage backend.
 
 TesseraCT is under active development, and will reach alpha in 2025Q3 🚀.
 
-At the moment, TesseraCT supports Amazon Web Service and Google Cloud Platform.
-Read the FAQ to understand [why we chose these platforms](#why-these-platforms),
-or if [you're interested in others](#can-i-run-tesseract-outside-gcp-or-aws).
+|Platform         |Architecture            |Our use-case          |Performance|Binary                                  |Deployment                                                      |
+|-----------------|------------------------|----------------------|-----------|----------------------------------------|----------------------------------------------------------------|
+|GCP              |Spanner + GCS + MIG     |public staging logs   |TODO       |[gcp](/cmd/tesseract/gcp/main.go)       |[doc](/deployment/live/gcp/static-ct-staging/logs/arche2025h1/) |
+|GCP              |Spanner + GCS + CloudRun|continuous integration|TODO       |[gcp](/cmd/tesseract/gcp/main.go)       |[example](/deployment/live/gcp/static-ct/logs/ci)               |
+|GCP              |Spanner + GCS + VM      |codelab, load tests   |TODO       |[gcp](/cmd/tesseract/gcp/main.go)       |[doc](/deployment/live/gcp/test/)                               |
+|AWS              |RDS + S3 + ECS          |continuous integration|TODO       |[aws](/cmd/tesseract/aws/main.go)       |[example](/deployment/live/aws/conformance/ci/)                 |
+|AWS              |RDS + S3 + VM           |codelab, load tests   |TODO       |[aws](/cmd/tesseract/aws/main.go)       |[doc](/deployment/live/aws/test/)                               |
+|POSIX            |ZFS + VM                |one-off test          |NA         |[posix](/cmd/experimental/posix/main.go)|[doc](/cmd/experimental/posix/)                                 |
```

Comment:
POSIX has the equivalent `codelab` in the README.

I'll add CI coverage and load-test info and we can update this row with those later.